cli/shared/src/test/scala/com/lightbend/rp/reactivecli/docker/DockerCredentialsTest.scala

```diff
@@ -40,10 +39,33 @@ object DockerCredentialsTest extends TestSuite {
            |""".stripMargin)
 
       val expected = Seq(
-        DockerCredentials("lightbend-docker-registry.bintray.io", "hello", "there"),
-        DockerCredentials("registry.hub.docker.com", "foo", "bar"),
-        DockerCredentials("2.hub.docker.com", "ok", "what"))
+        DockerCredentials("lightbend-docker-registry.bintray.io", "hello", "there", ""),
+        DockerCredentials("registry.hub.docker.com", "foo", "bar", ""),
+        DockerCredentials("2.hub.docker.com", "ok", "what", ""))
+
+      assert(result == expected)
+    }
+    "Parse Docker Config" - {
+      val result_empty = DockerCredentials.decodeConfig("""{"auths": {} }""")
```

Comment:
Bike shedding: `resultEmpty`, we're in camelCase land

Reply:
Good catch!